book/routing.rst

```diff
@@ -549,10 +549,11 @@ URL          Route     Parameters
 
 .. _book-routing-requirements:
 
-Adding Requirements
-~~~~~~~~~~~~~~~~~~~
+Route Requirements
+~~~~~~~~~~~~~~~~~~
 
-Take a quick look at the routes that have been created so far:
+Take a quick look at both routes that have been created so far for our
+imaginary blog::
```

Comment:
I don't think we need to add more to this sentence - it also uses the "first-person" "our" :/